.github/workflows/sanity-test.yaml

```diff
@@ -130,7 +132,7 @@ jobs:
 
           resultjson=$(sed -e '1,/Completed Backups/d' $TEST_RESULT/backup_exchange.txt ) 
 
-          if  [[ $( echo $resultjson | jq -r '.[0] | .errorCount') -ne 0 ]]; then
+          if  [[ $( echo $resultjson | jq -r '.[0] | .stats.errorCount') -ne 0 ]]; then
```

Comment:
@neha-Gupta1 - is this the fix for the unbounded variable error?

Reply:
Yes

Reply:
👍 - got it. Suggest we just push a separate PR for this ASAP just to get the tests moving. Also - maybe disable the failing test in that PR for now. That way we have a green run with less tests vs a red run with more tests.

The failing test can be re-enabled once the rest of this current PR merges.

Reply:
@vkamra, this is the PR for it - https://github.com/alcionai/corso/pull/3228

Reply:
I think GitHub is showing this as it is diffing this with an old version due to the fact that you have a merge conflict. If you could fix and rebase, I think we will have a better diff to review.

Reply:
Taken care of.